app/controllers/school_classes_controller.rb

```diff
@@ -0,0 +1,32 @@
+class SchoolClassesController < ApplicationController
+  def new
+		@school_class = SchoolClass.new
+	end
+
+  def create
+    @school_class = SchoolClass.new(post_params(:title, :room_number))
+    @school_class.save
+    redirect_to school_class_path(@school_class)
+  end
```

Comment:
Since you're not using the @school_class instance variable in a view, it could be a regular variable.